5.analyze-weights/scripts/latent.py

```diff
@@ -541,3 +544,51 @@ def run_gsea_pipeline_command(input_weight_dir, z_dim, dataset_name, num_perm,
     out_file = 'gsea_results_{}_zdim_{}.tsv'.format(dataset_name, z_dim)
     out_file = os.path.join('results', 'gsea', out_file)
     pd.concat(gsea_results_list).to_csv(out_file, sep='\t')
+
+
+def load_hetnets(hetnet_file, permuted_directory, subset_genes,
+                 metaedge_abbrev='GpXCELL'):
```

Comment:
Not sure if it might be useful to add a sentence about what hetnets are in the description for other users just in case

Reply:
Yep, it would be useful!